### PR TITLE
Sh skunkworks

### LINKED ISSUE #18: Add skunkworks pages
https://nhsx.github.io/skunkworks/

### PR DESCRIPTION
**Pull request by SamHollings**
# Description

❓**What**: Added the existing SKunkworks pages (https://nhsx.github.io/skunkworks/data-lens) and make them work. closes #18 

🧠**Why?**: They have merged with our team in the new NHSE.

👨‍💻**How?**: Dragged and dropped the pages, then made some slight edits so the images etc would work normally. This also needed me to edit some other bits of the site, such as the overrides, so it would pull stuff from the page metadata.
I haven't done a comprehensive check of the page contents - that's for the teams to do later.

# Checklist:
Have checked for the following:
- [x] The website still builds correctly, and you can view it using `mkdocs serve`.
- [x] There are no new "warnings" from mkdocs
- [x] Does your page follow the [page template](https://nhsdigital.github.io/rap-community-of-practice/example_RAP_CoP_page/) (or [here in Markdown](https://github.com/NHSDigital/rap-community-of-practice/blob/main/docs/example_RAP_CoP_page.md))? (**need to make a new one specific to NHSE Data Science**)
- [ ] Spelling errors
- [ ] Consistent capitalization
- [ ] Consistent numbers
- [ ] Material features incorrectly implemented: search for code blocks and markers (e.g. !!!).
- [ ] Code snippets don't work
- [ ] Images not working
- [ ] Links not working

## Where it was tested

- Github Codespaces - 2-core, 4GB RAM, 32GB hard drive
- devcontainer.json describes further settings

### CODE REVIEW

File: docs/our_work/adrenal-lesions.md
Comment (amaiaita) [inline docs/our_work/adrenal-lesions.md:1]: the tags in all of these pages are visible on the page in a weird format - i assume that wasnt the intention? 
Screenshot 2023-12-22 at 08 13 34

**Review by amaiaita — CHANGES_REQUESTED** [review]
If the tags and images (which I'm working on) is something we want to make work separately to this PR then feel free to merge! otherwise the tags need fixing :) looks good otherwise!

File: docs/our_work/adrenal-lesions.md
Comment (SamHollings) [reply docs/our_work/adrenal-lesions.md:1]: Yes, the tags are due to my own lack of skill!! that's the best I could do - I made [`./overrides/main.html`](https://github.com/nhsengland/datascience/blob/sh_skunkworks/overrides/main.html) take the tags from the metadata for the page... and then I didn't know how to do anything fancy with them! 

We can make that another ticket :)

Comment (SamHollings) [reply docs/our_work/adrenal-lesions.md:1]: I've made an issue #28

**Pull request merged**